packages/core/modules-sdk/src/medusa-app.ts

```diff
@@ -54,6 +54,7 @@ declare module "@medusajs/types" {
     [ContainerRegistrationKeys.CONFIG_MODULE]: ConfigModule
     [ContainerRegistrationKeys.PG_CONNECTION]: Knex<any>
     [ContainerRegistrationKeys.REMOTE_QUERY]: RemoteQueryFunction
```

Comment:
Currently we put the `query` inside the container as `REMOTE_QUERY`. Not sure if there as any way to rename it.

Maybe we document it anymore and use the new registration key `QUERY`?

Reply:
I believe we can just add a note that this will be removed, and a few releases later we can remove this key

Reply:
That can be done later, as we have to replace everywhere in the core

Reply:
Sure!